### DISCUSSION TITLE
Node filtering transactions

### DISCUSSION THREAD

**Discussion by mampudi** [ordinals/ord#1529] [Q&A] [answered]
What is the impact of nodes filtering these kinds of transactions?

https://twitter.com/oomahq/status/1621899175079051264?s=46&t=NNE0nFaVf5K2yEnadmoT9w

**Comment by casey** [comment, marked as answer]
Nothing really. If enough node operators ran it, it would make inscriptions hard to get to miners to be confirmed, but I doubt it'll ever become popular enough for it to be an issue. Especially since there are a lot of new ordinal degens running full nodes. By running a full node you help keep inscriptions moving quickly across the network. 🚅
